s4e-web/projects/notifications/src/lib/state/notification.service.spec.ts

```diff
@@ -0,0 +1,21 @@
+import {TestBed} from '@angular/core/testing';
+import {NotificationService} from './notification.service';
+import {NotificationStore} from './notification.store';
+import {NotificationsModule} from '../notifications.module';
+
+describe('NotificationService', () => {
+  let notificationService: NotificationService;
+  let notificationStore: NotificationStore;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [NotificationsModule.forRoot()],
+    });
+
+    notificationService = TestBed.get(NotificationService);
+    notificationStore = TestBed.get(NotificationStore);
+  });
+
+  it('TODO - redo tests', () => {
```

Comment:
What test case is it?

Reply:
This is the test case which had to be done (it should be good now)